airflow/www_rbac/app.py

```diff
@@ -59,10 +60,20 @@ def create_app(config=None, testing=False, app_name="Airflow"):
     configure_logging()
 
     with app.app_context():
+
+        from airflow.www_rbac.security import AirflowSecurityManager
+        custom_sm = app.config.get('CUSTOM_SECURITY_MANAGER') or AirflowSecurityManager
```

Comment:
why changing the key form `SECURITY_MANAGER_CLASS` to `CUSTOM_SECURITY_MANAGER `?

Reply:
will change back, may have been an oversight

Reply:
ignore my previous comment :)

Reply:
hey Joy @jgao54 , do you have any concern about the approach? If not, I will go ahead and update the pr per Max's comment and the todo list. Thanks a lot for helping the review :)